src/docfx/restore/PublicTemplatePackage.cs

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace Microsoft.Docs.Build
+{
+    internal class PublicTemplatePackage : Package
+    {
+        private readonly Uri _baseUrl;
+        private readonly FileResolver _fileResolver;
+
+        public PublicTemplatePackage(string baseUrl, FileResolver fileResolver)
+        {
+            _baseUrl = new Uri($"{baseUrl.TrimEnd('/')}/");
+            _fileResolver = fileResolver;
+        }
+
+        public override bool Exists(PathString path)
+        {
+            return _fileResolver.TryResolveFilePath(GetPath(path), out _);
```

Comment:
We should never hit this right?
```suggestion
    throw new NotSupportedException();
```

Reply:
it has been used when getting the schema

---
In reply to: [488341115](https://github.com/dotnet/docfx/pull/6550#discussion_r488341115) [](ancestors = 488341115)